tests/Unit/ControlSystem/Actions/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY_SOURCES
+  ${LIBRARY_SOURCES}
+  Actions/Test_InitializeMeasurements.cpp
```

Comment:
Could you also move the `Test_Initialization.cpp` into the `Actions` namespace in a separate commit. We should probably keep things consist.